### PR TITLE
131780323_methods_and_recursion_master

### PR DESCRIPTION
**Pull request by votillev717**

### CODE REVIEW

File: methods_and_recursion/bottles_of_beer.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!usr/bin/env ruby
```
Comment (jasonnoble) [inline methods_and_recursion/bottles_of_beer.rb]: `#!/usr/bin/env ruby` (missing a slash)

File: methods_and_recursion/bottles_of_beer.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+def bottles_of_beer
```
Comment (jasonnoble) [inline methods_and_recursion/bottles_of_beer.rb:3]: What's this method do?

File: methods_and_recursion/bottles_of_beer.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+number_of_bottles2 = 99
+
+until number_of_bottles < 2
+
+  number_of_bottles -= 1
+  number_of_bottles2 -= 1
+
+  puts "#{number_of_bottles} bottles of beer on the wall. "
+  puts "#{number_of_bottles} bottles of beer. Take one down pass it around. "
+  puts "#{number_of_bottles2} bottles of beer on the wall.
```
Comment (jasonnoble) [inline methods_and_recursion/bottles_of_beer.rb]: `puts "#{number_of_bottles2} bottles of beer on the wall.\n\n"`

**Review by jasonnoble — CHANGES_REQUESTED** [review]

File: methods_and_recursion/bottles_of_beer.rb (hunk 1 above)
Comment (votillev717) [reply methods_and_recursion/bottles_of_beer.rb]: Ok.

File: methods_and_recursion/bottles_of_beer.rb (hunk 2 above)
Comment (votillev717) [reply methods_and_recursion/bottles_of_beer.rb:3]: Not sure.

File: methods_and_recursion/bottles_of_beer.rb (hunk 3 above)
Comment (votillev717) [reply methods_and_recursion/bottles_of_beer.rb]: Ok.

**Pull request closed without merging**